cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -925,6 +925,16 @@ function Configure-GcePdTools {
 '$modulePath = "K8S_DIR\GetGcePdName.dll"
 Unblock-File $modulePath
 Import-Module -Name $modulePath'.replace('K8S_DIR', ${env:K8S_DIR})
+
+  if (Test-IsTestCluster $kube_env) {
+    if (ShouldWrite-File ${env:K8S_DIR}\diskutil.exe) {
+      // The source code of this executable file is https://github.com/kubernetes-sigs/sig-windows-tools/blob/master/cmd/diskutil/diskutil.c
```

Comment:
Powershell comments are #, not //

Reply:
oh, my mistake. fixed. thanks!

Reply:
I don't see the fix. Can you upload your latest?